rails/db/migrate/20191004155911_add_discarded_at_to_dives.rb

```diff
@@ -0,0 +1,6 @@
+class AddDiscardedAtToDives < ActiveRecord::Migration[6.0]
+  def change
+    add_column :dives, :discarded_at, :datetime
+    add_index :dives, :discarded_at
```

Comment:
I'm not sure if an index is necessary. Please have a look: https://stackoverflow.com/questions/26525068/does-laravels-soft-delete-need-index-on-mysql/26530488#26530488

If there is a reason, I would like to learn what is the reasoning behind that.

Reply:
Oh, just looked into gem readme! Interesting.